custom_components/miio_yeelink/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 from homeassistant import core, config_entries
 from homeassistant.const import *
-from homeassistant.config import DATA_CUSTOMIZE
+from homeassistant.core_config
```

Comment:
Needs to be `from homeassistant.core_config import DATA_CUSTOMIZE` instead

Reply:
> @lubosjerabek followed your instructions and now it works again. Thanks!